avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -1492,13 +1511,11 @@ private Schema toConnectSchema(org.apache.avro.Schema schema,
                   + " property must be a JSON Integer."
                   + " https://avro.apache.org/docs/1.7.7/spec.html#Decimal");
             }
+            // Capture the precision as a parameter only if it is not the default
             Integer precision = precisionNode.asInt();
-            builder.parameter(CONNECT_AVRO_DECIMAL_PRECISION_PROP, precision.toString());
-          } else {
-            builder.parameter(
-                CONNECT_AVRO_DECIMAL_PRECISION_PROP,
-                CONNECT_AVRO_DECIMAL_PRECISION_DEFAULT.toString()
-            );
```

Comment:
The old code is asymmetric -- it tolerates not having the `connect.decimal.precision=64` parameter but it always generates it. This makes comparing Connect `Schema` objects difficult before and after conversion to/from Avro. This change only writes this parameter if not equal to 64, so it's far more symmetric making `Schema.equals(...)` more useful more often.